anvil_schema/fss_index_data_dictionary/fss_index_data_dictionary.json

```diff
@@ -0,0 +1,888 @@
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "used_file_id", 
+          "description": "Unique ID used to identify the file or files used within an activity.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "reference_assembly", 
+          "description": "A reference to the collection of sequences taken as the standard for a given organism. May be defined by https://www.ncbi.nlm.nih.gov/grc.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        },
+        {
+          "name": "source_datarepo_row_ids", 
+          "description": "The identifiers of the Terra Data Repository rows that contributed to the creation of the record. For example, if row 123 in source table 'subject' was used to create row ABC in the 'anvil_donor' table, the source_datarepo_row_ids column for row ABC in 'anvil_donor' would contain 'subject:123'.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false}
+      ],
+      "primaryKey": ["alignmentactivity_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null,
+      "reqiured": false
+    },
+    {
+      "name": "anvil_antibody",
+      "description": "May be a reference to an antibody used for a SampleTreatmentActivity or another Activity.",
+      "columns": [
+        {
+          "name": "antibody_id", 
+          "description": "Unique ID used to identify the antibody represented in this record."
+          "datatype": "string", 
+          "array_of": false, 
+          "required": true
+        },
+        {
+          "name": "target", 
+          // couldn't find a definition - all references say "definition needed"
+          "description": ""
+          "datatype": "string", 
+          "array_of": false, 
+          "required": false
+        },
+        {
+          "name": "source_datarepo_row_ids", 
+          "description": "The identifiers of the Terra Data Repository rows that contributed to the creation of the record. For example, if row 123 in source table 'subject' was used to create row ABC in the 'anvil_donor' table, the source_datarepo_row_ids column for row ABC in 'anvil_donor' would contain 'subject:123'.",
+          "datatype": "string", 
+          "array_of": true, 
+          "required": false
+        }
+      ],
+      "primaryKey": ["antibody_id"],
+      "partitionMode": "none",
+      "datePartitionOptions": null,
+      "intPartitionOptions": null,
+      "reqiured": false
+    },
+    {
+      "name": "anvil_assayactivity",
+      /// not sure this makes sense
+      "description": "Activity table designed to capture information about assays related to files and biosamples.",
```

Comment:
"Experimental measurement" --> Doesn't create files/biosamples, creates values/tabular data

Reply:
Couldn't remember how to implement this feedback - can you take a look and see if I got it right?